src/storage/btree.c

```diff
@@ -15539,12 +15464,6 @@ btree_find_next_index_record (THREAD_ENTRY * thread_p, BTREE_SCAN * bts)
   first_page = bts->C_page;	/* init */
 
   ret_val = btree_find_next_index_record_holding_current (thread_p, bts, NULL);
-#if 0				/* TODO - need to check return value */
```

Comment:
Please leave it as before. It seems that we need to check the return value after analysis.